scheduler/README.md

```diff
@@ -0,0 +1,24 @@
+# coscheduler-same-node
+The repo is for trying to implements such a scheduler:
+https://github.com/tektoncd/pipeline/issues/3052
+
+# Installation
+```
+make image
+docker push
```

Comment:
FYI I'd like to get this to use `ko` to make it mesh easier with our existing build/release processes. This will make it easier to integrate into Tekton in the future if we seem the experiment a success.

Same with a lot of the stuff in hack/lib, since I'm not sure we really need it. That doesn't really affect the meat of this PR though, it's just something I'd like to do in the future.

Reply:
Make sense, thanks @ImJasonH , I will handle it.